tavern/_plugins/mqtt/response.py

```diff
@@ -38,8 +38,9 @@ def _get_payload_vals(self):
 
             if payload.pop("$ext", None):
                 raise exceptions.BadSchemaError(
-                    "$ext function found in block %s - this has been moved to verify_response_with block - see documentation",
-                    "json",
+                    "$ext function found in block {} - this has been moved to verify_response_with block - see documentation".format(
+                        "json",
+                    )
```

Comment:
- Worth a custom exception, since you use it 3 times?
- In another exception you link to the docs, that might be handy here too?